pkgs/by-name/li/libideviceactivation/package.nix

```diff
@@ -1,55 +1,57 @@
 {
+  lib,
   stdenv,
   fetchFromGitHub,
-  lib,
-  pkg-config,
+
   automake,
   autoreconfHook,
-  libtool,
-  libplist,
+  pkg-config,
+
+  curl,
   libimobiledevice,
+  libplist,
+  libtool,
   libxml2,
-  curl,
   usbmuxd,
 }:
-
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation {
```

Comment:
In this derivation self attributes aren't required, hence why I removed it. I don't mind returning it for consistency's sake, just wanted to point that out in case you missed it.

Reply:
I originally also had this without finalattrs and was told it should always be used.